crates/bevy_ui/src/flex/convert.rs

```diff
@@ -3,65 +3,113 @@ use crate::{
     PositionType, Size, Style, UiRect, Val,
 };
 
-pub fn from_rect(
-    scale_factor: f64,
-    rect: UiRect,
-) -> taffy::geometry::Rect<taffy::style::Dimension> {
-    taffy::geometry::Rect {
-        left: from_val(scale_factor, rect.left),
-        right: from_val(scale_factor, rect.right),
-        top: from_val(scale_factor, rect.top),
-        bottom: from_val(scale_factor, rect.bottom),
-    }
-}
-
-pub fn from_val_size(
-    scale_factor: f64,
-    size: Size,
-) -> taffy::geometry::Size<taffy::style::Dimension> {
-    taffy::geometry::Size {
-        width: from_val(scale_factor, size.width),
-        height: from_val(scale_factor, size.height),
-    }
-}
-
 pub fn from_style(scale_factor: f64, value: &Style) -> taffy::style::Style {
     taffy::style::Style {
         display: value.display.into(),
-        position_type: value.position_type.into(),
+        position: value.position_type.into(),
         flex_direction: value.flex_direction.into(),
         flex_wrap: value.flex_wrap.into(),
-        align_items: value.align_items.into(),
+        align_items: Some(value.align_items.into()),
```

Comment:
Not necessary for just flexbox, but for you grid I think you want the option of "unset" for align_items because the default varies (I think it's `Stretch` for relatively positioned items and `Start` for absolutely positioned items or something like that).

Reply:
I'll look into it. Maybe it's not worth making any breaking changes now and better to leave this for the grid implementation PR.

Reply:
I thought before it would be breaking because we'd need to make align_items optional, but just needs an `Unset` variant of course.

`AlignItems::Unset` acts like `Start` for absolute positions and `Stretch` for relative positions in my Bevy examples.

Reply:
Isn't the idea to move away from `Unset`/`Undefined` and use `Option` instead. It seems silly to remove `Undefined` from `Val` while simultaneously adding `Unset` here...

Reply:
Deleted the unset variant and kept align_items non-optional. Don't need to worry about this here, not implementing grid in this PR.